app/src/main/res/layout/view_image_detail.xml

```diff
@@ -27,16 +28,33 @@
         android:layout_height="1dp"
         android:background="?attr/chip_background_color" />
 
-    <TextView
-        android:id="@+id/detailTextView"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:id="@+id/detailsContainer"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="11dp"
-        android:fontFamily="sans-serif"
-        android:lineSpacingExtra="8sp"
-        android:textColor="?attr/primary_text_color"
-        android:textSize="16sp"
-        android:textStyle="normal"
-        tools:text="Watercolor portrait of Ada King, Countess of Lovelace (Ada Lovelace)" />
+        android:orientation="horizontal"
+        android:paddingTop="12dp">
```

Comment:
It would be better if add `android:background="?android:selectableItemBackground"` to this view

and please don't forget to add `paddingBottom` to make the text align to the middle. (and will need to adjust the parent's bottom padding)

Reply:
I did not want to align the external link icon to the center, because sometimes the details have a lot of new lines and that makes the icon look out of place.